### PR TITLE
Add DevOps Radar for November 2019

### PR DESCRIPTION
**Pull request by Starefossen**
Add all the relevant technologies for the November 2019 DevOps Radar. Please submit your suggestions or comments.

In this edition we won't take to much time on writing down our decision but in future versions we will require a short description for each tool and why we have placed it in the given ring. But we have to start somewhere 😄 

 The EVRY DevOps Radar is a list of technologies, complemented by an assessment result, called ring assignment. We use four rings with the following semantics:

*    **ADOPT** — Technologies we have high confidence in to serve our purpose, also in large scale. Technologies with a usage culture in our EVRY production environment, low risk and recommended to be widely used.
*    **TRIAL** — Technologies that we have seen work with success in project work to solve a real problem; first serious usage experience that confirm benefits and can uncover limitations. TRIAL technologies are slightly more risky; some engineers in our organization walked this path and will share knowledge and experiences.
*    **ASSESS** — Technologies that are promising and have clear potential value-add for us; technologies worth to invest some research and prototyping efforts in to see if it has impact. ASSESS technologies have higher risks; they are often brand new and highly unproven in our organisation. You will find some engineers that have knowledge in the technology and promote it, you may even find teams that have started a prototyping effort.
*    **HOLD** — Technologies not recommended to be used for new projects. Technologies that we think are not (yet) worth to (further) invest in. HOLD technologies should not be used for new projects, but usually can be continued for existing projects.

### CODE REVIEW

File: _data/radar.yaml [hunk 1]
```diff
@@ -1,13 +1,171 @@
...
+        link: https://octopus.com/
+        ring: 3
+
+      - name: Travis CI
+        link: https://travis-ci.com/
+        ring: 3
+
+      - name: Jenkins
+        link: https://jenkins.io/
+        ring: 4
```
Comment (bjorges) [inline _data/radar.yaml]: Moved this to 4, discuss if you disagree.

File: _data/radar.yaml [hunk 2]
```diff
@@ -1,13 +1,171 @@
...
+
+      - name: Fluentd
+        link: https://www.fluentd.org/
+        ring: 0
+
+      - name: cert-manager
+        link: http://docs.cert-manager.io/en/latest/
+        ring: 0
+
+      - name: Istio
```
Comment (bjorges) [inline _data/radar.yaml:182]: Why is this placed under Observability & Analysis? Should it be below security?

**Review by bjorges — COMMENTED** [review]
First time reviewing in GitHub. Be nice!

File: _data/radar.yaml (hunk 1 above)
Comment (Starefossen) [reply _data/radar.yaml]: Agreed!

File: _data/radar.yaml (hunk 2 above)
Comment (Starefossen) [reply _data/radar.yaml:182]: While Istio certainly has some security components to it, it is not primarily a security tool. Some tools may fall under multiple quadrants and we try to place them in their primary one. For guidance you can use the CNCF landscape for reference.

I don't have a particular strong opinion and if public's mindset of Istio is as a security tool we should move it. However there are multiple other service mesh products that are not as security focused as Istio hence why I chose to put it in Observability & Analysis.

**Review by bjorges — APPROVED** [review]

**Pull request merged**